### PR TITLE
Add trace and sql execution logs

### PR DESCRIPTION
**Pull request by yshan2028**
Add trace and sql execution logs and support writing to log files. Set the switch in the conf file. The default is off.

### CODE REVIEW

**Comment by wu-clan** [comment]
Hello, @yshan2028.

Thank you for the logging suggestion, for that reason, this is my take on it

1. you added two tools to the middleware directory, they are not really middleware, they are just tools
2. SQLAlchemy logging can be controlled by `MYSQL_ECHO` and have better output, including elapsed time, transactions, and more.
3. interface request logs are currently written to the database, for console and log files, just add `log.info()` to the operation log middleware
4. trace there are better ways, es, logfire, trace middleware

Our current logger is very powerful, if you are interested, try to understand it better: [log.py](https://github.com/fastapi-practices/fastapi_best_architecture/blob/master/backend/common/log.py)

To summarize, this PR may not meet expectations, and I think these logging features will increase the api time consuming.

**Comment by wu-clan** [comment]
If you're interested, check out our official implementation #409 #411 #413 and thanks again for your time!
